app/src/format.py

```diff
@@ -1,19 +1,36 @@
+import logging
+import random
 from typing import Sequence
 
 from src.db.models.document import ChunkWithScore
 
+logger = logging.getLogger(__name__)
+
 # We need a unique identifier for each accordion,
-# even across multiple calls to this function
-_accordion_id = 0
+# even across multiple calls to this function.
+# Choose a random number to avoid id collisions when hotloading the app during development.
+_accordion_id = random.randint(0, 1000000)
```

Comment:
Odds of collisions are low, but should we do a check for if the accordion id exists before assignment as well?